e2e/dractions/disapp.go

```diff
@@ -129,8 +129,13 @@ func failoverRelocateDiscoveredApps(
 		return err
 	}
 
+	currentDRCluster, err := env.GetEnvCluster(ctx, currentCluster)
```

Comment:
currentDRCluster is not the best name. The best name is currentCluster but it is used for the cluster names. Can we pass cluster objects instead of strings to this function?

A smaller change would be to rename the arguments to currentClusterName and targetClusterName.